src/pages/Admin/Venue/VenueDetailsSubForm.tsx

```diff
@@ -1,329 +1,49 @@
-import React, {
```

Comment:
Not a rename of `DetailsForm` file nor component. This one is the `DetailsFormLeft` component that previously existed in the file, but extracted in its own and renamed (since the right-side form isn't used due to `IS_BURN` flag being obsolete)